src/standalone/api/kernels/api.vscode.common.test.ts

```diff
@@ -185,6 +195,104 @@ suiteMandatory('Kernel API Tests @typescript', function () {
             'Traceback does not contain original error'
         );
     });
+    testMandatory('Kernel start event is not triggered by silent executions', async function () {
```

Comment:
Can we have another test for `kernel Start event is triggered when user executes code` or the like.
Or is that not possible?

Reply:
should now be covered by "Kernel start event is triggered when user executes code and the event execution runs first"